### PR TITLE
Add drop down menu for all selected rows

### PR DESCRIPTION
**Pull request by cptKNJO**
This PR adds the following feature:

    Add drop down to do the same with all selected rows

I am making a change to the API type in this PR. `ClientId` as array instead of tuple. Please let me know if my change is incorrect.

Thanks for reviewing!

(Also, please let me know if I can improve the PRs by, e.g. attaching screenshots.)

### CODE REVIEW

File: urbackupserver/www2/src/api/urbackupserver.ts [hunk 1]
```diff
@@ -297,7 +297,7 @@ class UrBackupServer {
   };
 
   // Start a backup of type `backupType` for client with id `clientId`
-  startBackup = async (clientId: [ClientIdType], backupType: BackupType) => {
+  startBackup = async (clientId: ClientIdType[], backupType: BackupType) => {
```
Comment (uroni) [inline urbackupserver/www2/src/api/urbackupserver.ts:300]: Meant to have an array type here anyway. Guess I never tested it with multiple clients...

File: urbackupserver/www2/src/features/status/StatusMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import {
```
Comment (uroni) [inline urbackupserver/www2/src/features/status/StatusMenu.tsx:1]: Nit: I'd have put those files into pages/status/components or similar (as long as they only are used by the page)

File: urbackupserver/www2/src/features/status/useBackupMutation.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { useMutation } from "react-query";
```
Comment (uroni) [inline urbackupserver/www2/src/features/status/useBackupMutation.ts:1]: Nit: Perhaps don't put just one function into one file (if there aren't any plans to add more functions etc. later)

**Comment by uroni** [comment]
Thanks!

File: urbackupserver/www2/src/features/status/StatusMenu.tsx (hunk 2 above)
Comment (cptKNJO) [reply urbackupserver/www2/src/features/status/StatusMenu.tsx:1]: Agreed, for localizing. Normally, `pages/` or `app/` only contains the actual screens in a React app i.e. anything you put in it is a valid route. 

But I don't think it's required here (yet). Should I add a commit to move content of `features/Status` into `pages/Status` after the final PR is accepted?

File: urbackupserver/www2/src/features/status/useBackupMutation.ts (hunk 3 above)
Comment (cptKNJO) [reply urbackupserver/www2/src/features/status/useBackupMutation.ts:1]: I normally keep custom React hooks in a single file, like React components (with related code unused elsewhere).

Of course, it depends on project preferences! Should I move these hooks to the single file in a future commit? (This commit uses the hook in two different places, but it's not the case in the final commit.)

**Pull request merged**